staging/src/k8s.io/api/apps/v1/types.go

```diff
@@ -237,6 +248,14 @@ type StatefulSetSpec struct {
 	// down. This requires the StatefulSetAutoDeletePVC feature gate to be enabled,
 	// which is alpha.  +optional
 	PersistentVolumeClaimRetentionPolicy *StatefulSetPersistentVolumeClaimRetentionPolicy `json:"persistentVolumeClaimRetentionPolicy,omitempty" protobuf:"bytes,10,opt,name=persistentVolumeClaimRetentionPolicy"`
+
+	// Ordinals controls how the stateful set creates pod and persistent volume
+	// claim names.
+	// The default behavior assigns a number starting with zero and incremented by
```

Comment:
Use lowercase for the field name.  Make this a paragraph (remove the newline above).  I might suggest instead of "how the stateful set creates" focus on the ordinal behavior, and then either here or in the 'templates' field describe how the name is generated with the ordinal by example.  A user who looks at this field should be able to understand the outcome.  I think templates is *slightly* better as a field (maybe there is a better one) but we should be explicit in the api doc how to achieve the desired outcome without users having to jump around.

Reply:
Added detail to the StatefulSetSpec.Template field, and an example on how pods are named.